misc-tools/dj_judgehost_cleanup.in

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+# @configure_input@
+#
+# This script can perform various cleanup tasks on a judgehost.
+
+# Abort when a single command fails:
+set -e
+
+JUDGINGDIR=$(readlink -fn @judgehost_judgedir@)
+PROGNAME=$(basename $0)
+
+usage()
+{
+	cat <<EOF
+Usage: $PROGNAME <command>
+Perform various cleanup tasks on a judgehost.
+
+Make sure that no judgings are in process while running this script.
+It is best to temporarily disable the judgehost in the web interface
+or stop the service.
+
+Commands:
+
+  mounts    Unmount stale bind-mounts to the chroot. These can appear
+              when then judgedaemon crashes in the middle of a judging.
```

Comment:
Could/should the judgedaemon (also) do this itself on startup?

Reply:
As this only happens after a judgedaemon crashed and it doesn't directly affect further judging, I'm a bit hesitant to add that to the judgedaemon. Also because the (un)mounting currently happens in `chroot-start-stop.sh`, which wouldn't really be the best place to add it.